android/build.gradle

```
@@ -66,17 +66,17 @@ afterEvaluate {
 }
 
 android {
-  compileSdkVersion safeExtGet("compileSdkVersion", 34)
+  compileSdkVersion safeExtGet("compileSdkVersion", 31)
```

We can bump this to 33 at the very least
```suggestion
  compileSdkVersion safeExtGet("compileSdkVersion", 33)
```